charts/vela-core/templates/defwithtemplate/annotations.yaml

```diff
@@ -1,10 +1,10 @@
 # Code generated by KubeVela templates. DO NOT EDIT. Please edit the original cue file.
-# Definition source cue file: vela-templates/definitions/internal/annotations.cue
+# Definition source cue file: vela-templates/definitions/deprecated/annotations.cue
 apiVersion: core.oam.dev/v1beta1
 kind: TraitDefinition
 metadata:
   annotations:
-    definition.oam.dev/description: Add annotations on K8s pod for your workload which follows the pod spec in path 'spec.template'.
+    definition.oam.dev/description: Add annotations on K8s pod for your workload which follows the pod spec in path 'spec.template'. This definition is DEPRECATED, please specify annotations in component instead.
```

Comment:
Why is `annotation` deprecated? I knew two guys who were very happy for this feature a few months ago?

Reply:
+1 , should be be deprecated